src/components/DateTimePicker/DatePicker/Styles/mixins.scss

```diff
@@ -61,8 +61,12 @@ $picker-input-padding-vertical: max(
 
   &::-webkit-scrollbar {
```

Comment:
what happens for non webkit browsers?

Reply:
Good catch, will add Mozilla support prior to merge, at the moment it defaults to the browser scrollbars, doesn't look too bad, but may be improved a bit.

Reply:
Mozilla doesn't support custom scrollbars